DLL_Project/Toils/Toils_FoodSynthesizer.cs

```diff
@@ -58,7 +58,37 @@ private static Toil TakeFromSynthesier( TargetIndex ind, Pawn eater, Func<ThingD
 
         public static Toil TakeDrugFromSynthesizer( TargetIndex ind, Pawn eater )
         {
-            return TakeFromSynthesier( ind, eater, FoodSynthesis.IsDrug, FoodSynthesis.SortDrug );
+            CCL_Log.Message("Take from drugs.");
+            //would like to put this in the FoodSynthesis class but this would require changing the allowed function
+            //signature to take pawn as well. And a full refactor of all of the functions.

```

Comment:
If a full refactor is required to make it work then it may be worth it. Currently no public mods use this functionality anyway so no mods will be losing anything.